lib/praxis/response.rb

```diff
@@ -18,6 +18,26 @@ def self.inherited(klass)
       klass.status = self.status if self.status
     end
 
+    def self.generate_example( route: , headers: , headers_example: , payload_type: , payload_example: )
+      binding.pry
+    end
+
+    def self.url_example(route:, example_hash:{}, params: )# TODO: what context to use?, context: [r.id])
```

Comment:
isn't this the same exact function that exists in ActionDefinition? let's merge them